bundles/core/org.eclipse.smarthome.core.persistence/META-INF/MANIFEST.MF

```diff
@@ -6,8 +6,20 @@ Bundle-Version: 0.8.0.qualifier
 Bundle-ManifestVersion: 2
 Bundle-License: http://www.eclipse.org/legal/epl-v10.html
 Bundle-SymbolicName: org.eclipse.smarthome.core.persistence
-Import-Package: org.eclipse.smarthome.core.items,
+Import-Package: io.swagger.annotations,

```

Comment:
please mark this as an optional import, so that it only has to be present at compile time, but not runtime.